### PR TITLE
Fixed #2139 - Snackbar is displayed only once while bookmarking

### LINKED ISSUE #2139: unexpected behavior of snackbar when bookmarking
**Actual Behaviour**

When we bookmark an event in Schedule fragment sometimes the snackbar is shown twice with the message "bookmark added".

**Expected Behaviour**

The bookmark should only be shown once.

**Steps to reproduce it**

open the app, select the schedule fragment from the navigation bar and when we select the bookmark icon the snackbar must be shown once.

**gif of the issue**
open

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by raj10071997**
Fixes #2139

Changes: Since there is already a subscriber which shows the snackbar with "bookmark added" message on successful completion of the completable, there is no need to show the snackbar message again. So removed that line which called the snackbar again.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@raj10071997 Please resolve conflicts

File: android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java [hunk 1]
```diff
@@ -123,10 +123,9 @@ public void bindSession(RealmDataRepository realmRepo) {
                             });
 
                     realmRepo.setBookmark(sessionId, true).subscribe();
+
                     slotBookmark.setImageResource(R.drawable.ic_bookmark_white_24dp);
                     slotBookmark.setColorFilter(storedColor, PorterDuff.Mode.SRC_ATOP);
-                    if (onBookmarkSelectedListener != null)
-                        onBookmarkSelectedListener.showSnackbar(new BookmarkStatus(storedColor, sessionId, CODE_UNDO_ADDED));
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java:129]: Are you sure this line needs to be deleted?

Comment (raj10071997) [reply android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java:129]: Yes , I have checked it locally in my phone.

Comment (raj10071997) [reply android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java:129]: @iamareebjamal I have updated the branch. Please review it again and merge it. If there are some other changes to be done, tell me.

Comment (iamareebjamal) [reply android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java:129]: I am AFK. I'll do it ASAP

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**